documentation/user-consent.md

```diff
@@ -48,122 +43,269 @@ The document details aspects regarding the consent management, which includes fo
 
 ## User Authentication/Authorization & Consent Management
 
->[TO BE EDITED/COMPLETED] - The base standard is OpenID Connect. User authentication: Verify that the user is who they say they are. For example: IP/Network authentication, SMS OTP, User/Password, Push to operator application... it depends on the type of authorization (grant type) chosen according to the application use cases, integration model and requirements (B2B/B2C, via aggregator or not, delegated consent or not...). User authorization: Verify access to the requested resource (OpenID Connect scope, purpose) through policies and rules... user consent is checked at this point as a pre-requisite before issuing an access token.
+>[TO BE EDITED/COMPLETED] - The base standard is OpenID Connect (OIDC). <br>User authentication: Verify that the user is who they say they are. For example: IP/Network authentication, SMS OTP, User/Password, Push to operator application... it depends on the type of authorization (grant type) chosen according to the application use cases, integration model and requirements (B2B/B2C, via aggregator or not...).<br>User authorization: Verify access to the requested resource (OpenID Connect scope, purpose) through policies and rules... user consent is checked at this point as a pre-requisite before issuing an access token.
 
 ### Authentication mechanism/s
 
 >[TO BE EDITED/COMPLETED] - Authentication mechanism/s to be considered... E.g.: IP/Network authentication, SMS OTP, User/password, Push to operator application​...
 
 ### Consent capture and storage
 
->[TO BE EDITED/COMPLETED] - Consent capture (by operator vs by API Client a.k.a delegated) and storage (in operator vs in a third party).
+>[TO BE EDITED/COMPLETED] - Consent capture by operator and storage in operator (consent master).
 
 ### Authorization flows / grant types
 
->[TO BE EDITED/COMPLETED] - Authorization flows / grant types to be considered... E.g.: Authorization code flow, JWT-bearer flow, CIBA flow, Client credentials flow. For each API it should be possible to categorize it depending on the use case, integration/GTM model, personal information processing requirements, etc... and define the grant type/s to be used for each API.
-
-#### Authorization code flow
+#### Authorization code flow (Frontend flow)
 
->[TO BE EDITED/COMPLETED] - Flow to obtain an access token to consume network API...
+```mermaid
+sequenceDiagram
+autonumber
+title Consume a CAMARA API - Authorization Code Grant (FrontEnd)
+participant FE as Device App
+participant BE as Invoker (App Backend/Aggregator)
+box Operator
+  participant ExpO as OGW Platform<br>@Operator
```

Comment:
I suggest to remove  references to OGW in and rename _OGW Platform_ to _API Gateway Platform_ or _CAMARA API Gateway Platform_.  Diagrams in Authentication and Authorization Concept for Service APIs[](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-AuthN-AuthZ-Concept.md) use _Gateway_ node.
```suggestion
  participant ExpO as API Gateway<br> Platform @Operator
```

Reply:
+1 I missed to remove OGW references in the doc. Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/54/commits/c7f9715fa8c597e4c9f94e8f0bbf3aecc501ead5. 

Finally, I used the term "API Exposure Platform" because that's what was agreed upon in the past. And because, in general, the exposure platform could be not only the API GW, but also the Auth Server, etc. I have also included the definition of the term in the Glossary section. For clarity.